actions.yaml

```diff
@@ -1,6 +1,20 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
+anonymize-user:
+  description: |
+    Anonymizes an user in order to make it GDPR compliant.
+  properties:
+    username:
+      description: |
+        User name to be anonymized.
+      type: string
+    admin:
```

Comment:
You don't really need this for the API call. Perhaps you can find a  way to instantiate a user without this

Reply:
I highly doubt that, since the class has admin as a required parameter and runs pydantic valiation on all the fields.

Reply:
How about adding a default "false" value to that arg in the constructor or make the attr optional?